tests/data/Receiver01_2019_11_26_040_to_200MHz/25C/S11/AntSim4/Short02.s1p

```diff
@@ -0,0 +1,10 @@
+# Hz S RI R 50
+40000000	0.239204281177602	0.934098880926403
+40250000	0.249599052839829	0.931344542242144
+40500000	0.259930090010927	0.928471693086805
+40750000	0.270258408789301	0.925457604404284
+41000000	0.28054512636555	0.92236495470031
+41250000	0.290796746439387	0.919137761729966
+41500000	0.300995335565232	0.915797505494576
+41750000	0.3111560547094	0.912360402525328

```

Comment:
Spacing is off here, not sure if it will mess with tests.

Reply:
Oh, weird! There are no tests that actually read the files yet, I don't think. I guess we'll keep it in mind for when there are.